ggml/src/ggml-sycl/element_wise.cpp

```diff
@@ -1432,6 +1486,12 @@ void ggml_sycl_gelu_quick(ggml_backend_sycl_context & ctx, ggml_tensor * dst) {
     GGML_SYCL_DEBUG("call %s done\n", __func__);
 }
 
+void ggml_sycl_gelu_erf(ggml_backend_sycl_context & ctx, ggml_tensor * dst) {
+    GGML_SYCL_DEBUG("call %s: DST Tensor type: %s\n", __func__, ggml_type_name(dst->type));
```

Comment:
I just merged my dbg print PR, do you mind rebasing and using `scope_op_debug_print`?